src/migrations.js

```diff
@@ -35,6 +35,21 @@ exports.getPendingJobs = async () => {
   return pendingMigrations
 }
 
+exports.runPendingJobs = async () => {
+  const config = await getConfig()
+
+  const pendingJobs = await this.getPendingJobs()
```

Comment:
Wow, now I feel mighty dumb. Today I learned 2 things!

Of course `this` can be a substitute for `exports` thanks to the magic of arrow functions.How have I not realized this before?! :D

To verify the behavior I wrote [a few quick tests](https://gist.github.com/Sleavely/73a11686fc49bcb10dc55a37b0aa9c72) just to verify that it works the same way as exports, and also had the epiphany that when you're testing/mocking methods within the same file you can simply use `jest.spyOn` to toggle back and forth between mock behavior and real behavior. I don't know why that never occurred to me.